R/tm_outliers.R

```diff
@@ -1217,7 +1269,7 @@ srv_outliers <- function(id, data, reporter, filter_panel_api, outlier_var,
 
     teal.widgets::verbatim_popup_srv(
       id = "rcode",
-      verbatim_content = reactive(teal.code::get_code(final_q())),
+      verbatim_content = reactive(teal.code::get_code(req(final_q()))),
```

Comment:
The reporter also needs updating, line 1288

I've used the `req()` as well just to be safe, do you think it's necessary?

Reply:
@averissimo `req` is needed because of this comment (there is an example code to run to see the issue)
https://github.com/insightsengineering/teal.modules.general/pull/795/files#r1848069471
However this needs to be tested, after we fixed reactivity in here. Maybe is not needed anymore